dz12_ex3.py

```diff
@@ -0,0 +1,16 @@
+#3. В попередньо написаний кастомний Exception додати запис помилки і час її виникнення у файл.
+
+import time
+
+try:
+    # Код в якому ловимо помилки
+    # k = 5 / 0
+    k = 5+'dfs'
+    # print(my_list[gf])
+
+except Exception as e:
```

Comment:
You should log your custom exception usage. To do it, write to a file inside of the __init__ method of your exception:
```
class MyCustomException(Exception):
    def __init__(self, message):
        # write to a file here